torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -180,7 +180,7 @@ def __init__(self, gm, skip_constructors=False):
 
     def insertable_tensor_check(self, t: torch.Tensor) -> bool:
         # TODO - we could also Tensors which get replaced with arange here
-        return (
+        return bool(
```

Comment:
why do we need this cast?

Reply:
Sounds like there is a typing bug with one of the logical and overloads.

Reply:
`Tensor.all()` below returns a Tensor rather than a `bool`. I'll move this bool cast to that line to make things clearer